code/docker/c-lightning-base/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:18.04
```

Comment:
we use bitcoind's prebuilt docker image but we build lightningd image by ourselves. will it be a good idea to also use   c-lightning/lnd's docker image?

lnd's: https://github.com/lightningnetwork/lnd/blob/master/docker/README.md

it also provide a good tutorial on how nodes interact with each other, and [how to run the image](https://github.com/lightningnetwork/lnd/blob/master/docs/DOCKER.md)

Reply:
I just search.   seems  there's no official  docker images of   c-lightning /  lnd    published on docker hub.  I believe that's the reason @aantonop choose to build it by ourselves.

alternatives approaches can be:

+ use https://hub.docker.com/r/cdecker/lightning-ci/dockerfile
+ or maybe @Roasbeef  can publish a docker image onto docker hub on behalf of lightninglabs?